### DISCUSSION TITLE
Prevent multiple instances of the same feed from being added

### DISCUSSION THREAD

**Discussion by shooftie** [SamTV12345/PodFetch#426] [Ideas]
I was having trouble uploading my OPML and have managed to add 5 instances of many of the feeds.

It would be helpful to prevent the same feed from being added multiple times, additionally, it would be even more helpful to throw up a notification that this was the case.

![image](https://github.com/SamTV12345/PodFetch/assets/2161844/59ec99f8-dbea-4630-ab34-768d278e20a9)

**Comment by SamTV12345** [comment]
@shooftie Can you upload the OPML here so I can fiddle around?

**Reply by shooftie** [reply]
Thanks mate, I hope you don't mine me opening so many threads. I am really impressed by PodFetch so far. There are one or two features that are preventing me from being able to use it fully but considering the rate of development so far, I'm sure that it won't take long.

**Comment by shooftie** [comment]
Sure thing.

Just to be clear, I was having trouble uploading when I was using the old Postgres image; the upload process seemed to hang and then crash part way through. Each time I restarted the process it created a new instance of some of the feeds with a number appended to the directory name.

[export.opml.zip](https://github.com/SamTV12345/PodFetch/files/13369425/export.opml.zip)

This OPML contains all of my feeds, some of which are now dead.
